packages/stg-evm-usdt-v2/test/helpers/signPermit.js

```diff
@@ -0,0 +1,37 @@
+const { fromRpcSig } = require('ethereumjs-util');
+
+const EIP2612_TYPE = [
+    { name: 'owner', type: 'address' },
+    { name: 'spender', type: 'address' },
+    { name: 'value', type: 'uint256' },
+    { name: 'nonce', type: 'uint256' },
+    { name: 'deadline', type: 'uint256' },
+];
+
+module.exports = async ({ chainId, account, spender, version, tokenName, tokenAddress, amount, deadline }) => {
```

Comment:
`module.exports` is the old syntax, once you convert this to a TypeScript file you can use `export default`